### PR TITLE
171 feat i18n in application

### LINKED ISSUE #171: feat: i18n in application

### PR DESCRIPTION
**Pull request by Thechi2000**
Adds internationalization for static content. Locale can be selected in the new Accessibility view, and is stored in the `news_settings` table (to be used later for the generation). Translations are stored in `lib/l10n`.

Currently only features English and French locales.

Also fixes a bug in the provider menu, causing it to close even when there were errors on platforms other than Android, and removes all debug prints.

Closes #171

### CODE REVIEW

**Review by pmdlt — COMMENTED** [review]
Absolutely nothing to complain about, the code is clean and well structured. The application compiles well, just like before, but with the second language as well - top!

I've checked for any layout problems related to the French language, but it's all good! 

However, a few translations are missing, so I've put together a small, non-exhaustive selection. I invite a second reviewer to double-check!

- In the news view (the widget probably) there's a `Read More` link that isn't translated
- Fetch news button not translated
- Alarm view not translated

Alerts seem to have been broken too.

---

That's it, it's not blocking in itself so I didn't make any request changes, if you tell me it's ok/done in another PR then I'll approve without any problem.

File: lib/views/accessibility.dart [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+              child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    Text(
+                      loc.accessibilityLanguage,
+                      style: Theme.of(context).textTheme.bodyLarge,
+                    ),
+                    DropdownButton(
+                      value: loc.localeName,
+                      items: AppLocalizations.supportedLocales
```
Comment (pmdlt) [inline lib/views/accessibility.dart:37]: I couldn't find the exact place where the names of the languages were defined, but `français` should be capitalized.

Comment (Thechi2000) [reply lib/views/accessibility.dart:37]: It was handled by an external library, but I removed it to have more control over it.

**Review by pmdlt — DISMISSED** [review]
Thanks for all the changes!

**Comment by Thechi2000** [comment]
Thanks for the comments ! I added the missing translations and fixed the locales' display name. About the alerts, is it related to this PR ? It is working fine on my Android.

**Review by pmdlt — APPROVED** [review]

**Review by quartztz — APPROVED** [review]
No comment to be made; flawless code all around ! Thank you for the feature, can be merged asap :D

**Pull request merged**